cmd/unused-exporter/exporter.go

```diff
@@ -286,10 +291,12 @@ func lastUsedTS(d unused.Disk) float64 {
 }
 
 func getRegionFromZone(p unused.Provider, z string) string {
-	if strings.ToLower(p.Name()) == "azure" {
+	switch p.Name() {
+	case providerAWS :
+		return z[:len(z)-1]
+	case providerGCP:
+		return z[:len(z)-2]
```

Comment:
In #93 @jjo introduced a change that allow the user to change the provider name. This would break this switch and the if in line 269. I'd recommend instead doing the following:
```suggestion
	case aws.ProviderName:
		return z[:len(z)-1]
	case gcp.ProviderName:
		return z[:len(z)-2]
```

Note that by doing this and changing the if in 269 accordingly, you can get rid of the new constants in line 15.